diff --git a/node_modules/whatsapp-web.js/src/structures/Chat.js b/node_modules/whatsapp-web.js/src/structures/Chat.js
--- a/node_modules/whatsapp-web.js/src/structures/Chat.js
+++ b/node_modules/whatsapp-web.js/src/structures/Chat.js
@@ -206,7 +206,7 @@

             if (searchOptions && searchOptions.limit > 0) {
                 while (msgs.length < searchOptions.limit) {
-                    const loadedMessages = await window.Store.ConversationMsgs.loadEarlierMsgs(chat,chat.msgs);
+                    const loadedMessages = await window.require('WAWebChatLoadMessages').loadEarlierMsgs({ chat });
                     if (!loadedMessages || !loadedMessages.length) break;
                     msgs = [...loadedMessages.filter(msgFilter), ...msgs];
                 }
diff --git a/node_modules/whatsapp-web.js/src/structures/Channel.js b/node_modules/whatsapp-web.js/src/structures/Channel.js
--- a/node_modules/whatsapp-web.js/src/structures/Channel.js
+++ b/node_modules/whatsapp-web.js/src/structures/Channel.js
@@ -303,7 +303,7 @@

             if (searchOptions && searchOptions.limit > 0) {
                 while (msgs.length < searchOptions.limit) {
-                    const loadedMessages = await window.Store.ConversationMsgs.loadEarlierMsgs(channel);
+                    const loadedMessages = await window.require('WAWebChatLoadMessages').loadEarlierMsgs({ chat: channel });
                     if (!loadedMessages || !loadedMessages.length) break;
                     msgs = [...loadedMessages.filter(msgFilter), ...msgs];
                 }
