diff --git a/node_modules/whatsapp-web.js/src/structures/Chat.js b/node_modules/whatsapp-web.js/src/structures/Chat.js
--- a/node_modules/whatsapp-web.js/src/structures/Chat.js
+++ b/node_modules/whatsapp-web.js/src/structures/Chat.js
@@ -206,7 +206,7 @@

             if (searchOptions && searchOptions.limit > 0) {
                 while (msgs.length < searchOptions.limit) {
-                    const loadedMessages = await window.Store.ConversationMsgs.loadEarlierMsgs(chat,chat.msgs);
+                    const loadedMessages = await window.require('WAWebChatLoadMessages').loadEarlierMsgs({ chat });
                     if (!loadedMessages || !loadedMessages.length) break;
                     msgs = [...loadedMessages.filter(msgFilter), ...msgs];
                 }
diff --git a/node_modules/whatsapp-web.js/src/structures/Channel.js b/node_modules/whatsapp-web.js/src/structures/Channel.js
--- a/node_modules/whatsapp-web.js/src/structures/Channel.js
+++ b/node_modules/whatsapp-web.js/src/structures/Channel.js
@@ -303,7 +303,7 @@

             if (searchOptions && searchOptions.limit > 0) {
                 while (msgs.length < searchOptions.limit) {
-                    const loadedMessages = await window.Store.ConversationMsgs.loadEarlierMsgs(channel);
+                    const loadedMessages = await window.require('WAWebChatLoadMessages').loadEarlierMsgs({ chat: channel });
                     if (!loadedMessages || !loadedMessages.length) break;
                     msgs = [...loadedMessages.filter(msgFilter), ...msgs];
                 }
diff --git a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
--- a/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
+++ b/node_modules/whatsapp-web.js/src/util/Injected/Utils.js
@@ -9,17 +9,10 @@ exports.LoadUtils = () => {
         return await window.Store.ForwardUtils.forwardMessages({'chat': chat, 'msgs' : [msg], 'multicast': true, 'includeCaption': true, 'appendedText' : undefined});
     };

-    window.WWebJS.sendSeen = async (chatId) => {
-        const chat = await window.WWebJS.getChat(chatId, { getAsModel: false });
-        if (chat) {
-            window.Store.WAWebStreamModel.Stream.markAvailable();
-            await window.Store.SendSeen.sendSeen(chat);
-            window.Store.WAWebStreamModel.Stream.markUnavailable();
-            return true;
-        }
-        return false;
-    };
-
+    window.WWebJS.sendSeen = async (chatId) => {
+        // disabled to prevent markedUnread crash
+        return true;
+    };
     window.WWebJS.sendMessage = async (chat, content, options = {}) => {
         const isChannel = window.Store.ChatGetters.getIsNewsletter(chat);
         const isStatus = window.Store.ChatGetters.getIsBroadcast(chat);
